src/main/java/core/basesyntax/PrintInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface PrintInfo {
+    String getInformation();
```

Comment:
```suggestion
public interface Drawable {
    void draw();
```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/ec3819b6-8256-4417-a608-b31845531f53)